src/globalView.js

```diff
@@ -0,0 +1,2179 @@
+const webglUtils = require('./webgl-utils.js');
+const libUtility = require('./utility.js');
+const libTextRenderContext = require('./textRenderContext.js');
+const libPointViewer = require('./pointViewer.js');
+const libImageViewer = require('./imageViewer');
+const libDensityViewer = require('./densityViewer.js');
+const libHistogramViewer = require('./histogramViewer.js');
+const libCoordinateSystem = require('./coordinateSystem.js');
+const libColormap = require('./colormap.js');
+const libAlgorithm = require('./algorithm.js');
+const libGraphics = require('./graphics.js');
+const libGlMatrix = require('gl-matrix');
+
+/*
+function myAlert(msg) {
+  alert(msg); // eslint-disable-line no-alert, no-undef
+}
+
+export function initCanvas(canvasElement) {
+  // var canvas = document.getElementById("canvas");
+  const gl = canvasElement.getContext('webgl') ||
+    canvasElement.getContext('experimental-webgl');
+
+  // Only continue if WebGL is available and working
+  if (!gl) {
+    myAlert('Unable to initialize WebGL. Your browser or machine may not support it.');
+    return;
+  }
+
+  gl.clearColor(1, 0, 0, 1);
+  gl.clear(gl.COLOR_BUFFER_BIT | gl.DEPTH_BUFFER_BIT); // eslint-disable-line no-bitwise
+}
+*/
+
+// >>> Options
+
+var ENABLE_CONTINUOUS_RENDERING = false;
+var SHOW_FPS = false;
+var SIMULATE_LOW_FPS = false;
```

Comment:
I'm guessing this stuff was for the benchmarking and examples? Can delete if so.

Reply:
These are to show the frame rate in continuous rendering. I am not sure if we want to remove them as it won't be trivial to compute fps from outside example codes.